### DISCUSSION TITLE
How to get the Jacobian and Hessian value after training the neural network?

### DISCUSSION THREAD

**Discussion by DOUBLEYANG-Github** [lululxvi/deepxde#1451] [Q&A]
I know that there is a method in deepXDE: [https://deepxde.readthedocs.io/en/latest/modules/deepxde.html#module-deepxde.gradients](url) that calculates the Jacobian matrix when training the neural network. Suppose I have the trained network, how do I get the corresponding Jacobian given an input value?

**Comment by Dkupetsk** [comment]
I have the same question

**Comment by lululxvi** [comment]
See https://deepxde.readthedocs.io/en/latest/demos/pinn_forward/poisson.1d.dirichlet.html
```
model.predict(x, operator=pde)
```
